apiserver/src/test/java/org/dependencytrack/ResourceTest.java

```diff
@@ -135,7 +136,7 @@ public void after() {
         // code base can leave such a broken state behind if they run into unexpected
         // errors. See: https://github.com/DependencyTrack/dependency-track/issues/2677
         if (!qm.getPersistenceManager().isClosed()
-            && qm.getPersistenceManager().currentTransaction().isActive()) {
+                && qm.getPersistenceManager().currentTransaction().isActive()) {
```

Comment:
```suggestion
         && qm.getPersistenceManager().currentTransaction().isActive()) {
```

Not sure if this suggestion will fix it, but I'm guessing this is a typo

Reply:
```suggestion
            && qm.getPersistenceManager().currentTransaction().isActive()) {
```
Still seems like an unnecessary formatting change since we're not touching this line otherwise. my initial comment intention was to remove any changes to this section of code, not really a formatting preference